src/eq.js

```diff
@@ -1,4 +1,5 @@
-import React, { Component, cloneElement } from 'react';
+import { Component, cloneElement } from 'react';
+import PropTypes from 'prop-types';
```

Comment:
Do we also need to include `prop-types` as one of the dependencies in the `package.json`?